### PR TITLE
fix(bench): added group banner to bench output.

### PR DESCRIPTION
**Pull request by raashidanwar**
## Description
Added group banner to bench output.

After the changes for the given file, the console output should come.

```javascript
Deno.bench("url parsing", { group: "url", baseline: true }, () => {
    new URL("https://deno.land")
});

Deno.bench("url parsing with a referrer", { group: "url" }, () => {
    new URL("./welcome.ts", "https://deno.land/");
});

const BASE_URL = new URL("https://deno.land/");
Deno.bench("url parsing with a referrer URL obj", { group: "url" }, () => {
    new URL("./welcome.ts", BASE_URL);
});

const TEXT = "Lorem ipsum dolor sit amet";
Deno.bench("split on whitespace", { group: "text" }, () => {
    TEXT.split(" ");
});
```
output:
Screenshot 2023-12-13 at 9 07 49 AM

## Issue
- https://github.com/denoland/deno/issues/21451

### CODE REVIEW

File: cli/tests/testdata/bench/multifile_summary.out [hunk 1]
```diff
@@ -37,6 +39,7 @@ bench7 [WILDCARD] [WILDCARD]/iter[WILDCARD]([WILDCARD] … [WILDCARD]) [WILDCARD
 bench8 [WILDCARD] [WILDCARD]/iter[WILDCARD]([WILDCARD] … [WILDCARD]) [WILDCARD]
 bench9 [WILDCARD] [WILDCARD]/iter[WILDCARD]([WILDCARD] … [WILDCARD]) [WILDCARD]
 
+
 [WILDCARD]/bench/group_baseline.ts
 benchmark           time (avg)        iter/s             (min … max)       p75       p99      p995
 -------------------------------------------------------------------- -----------------------------
```
Comment (bartlomieju) [inline cli/tests/testdata/bench/multifile_summary.out:45]: Could you modify this test to use two different groups (ie. in addition to `url` group, add `noop` group).

File: cli/tools/bench/reporters.rs [hunk 2]
```diff
@@ -178,21 +178,14 @@ impl BenchReporter for ConsoleReporter {
...
-
         if self.group.is_none() || group != self.group.as_ref().unwrap() {
           self.report_group_summary();
         }
 
         if (self.group.is_none() && self.has_ungrouped)
           || (self.group.is_some() && self.group_measurements.is_empty())
         {
-          println!();
+          println!("group {}", colors::green(group));
```
Comment (bartlomieju) [inline cli/tools/bench/reporters.rs:188]: Slight nitpick, but `group` is not super relevant, so maybe print it using `colors::gray` so it shows as less significant that the actual bench name

**Review by bartlomieju — COMMENTED** [review]
Looks good, nice work @raashidanwar. Could you please apply these two suggestions?

File: cli/tools/bench/reporters.rs (hunk 2 above)
Comment (raashidanwar) [reply cli/tools/bench/reporters.rs]: Done

File: cli/tests/testdata/bench/multifile_summary.out (hunk 1 above)
Comment (raashidanwar) [reply cli/tests/testdata/bench/multifile_summary.out]: Done.

**Review by bartlomieju — APPROVED** [review]
LGTM, nice work, thank you @raashidanwar!

**Pull request merged**